_mobile/android.md

```diff
@@ -312,16 +313,15 @@ long[] getDataAsLongArray()
 float[] getDataAsFloatArray()
 double[] getDataAsDoubleArray() 
 ```
-These methods throw IllegalStateException if called for inappropriate dtype.
+These methods throw `IllegalStateException` if called for inappropriate dtype.
 
-### org.pytorch.IValue (IValue)
-[github](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/IValue.java)
+### [`org.pytorch.IValue`](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/IValue.java)
 
-IValue represents a TorchScript variable that can be one of the supported (by torchscript) types ( https://pytorch.org/docs/stable/jit.html#types ). IValue is a tagged union. For every supported type it has a factory method, method to check the type and a getter method to retrieve a value.
-Getters throw IllegalStateException if called for inappropriate type.
+IValue represents a TorchScript variable that can be one of the supported (by torchscript) types ( [https://pytorch.org/docs/stable/jit.html#types](https://pytorch.org/docs/stable/jit.html#types) ). 
```

Comment:
hyperlink to the word types
[types](https://pytorch.org/docs/stable/jit.html#types)